operation/manager_test.go

```diff
@@ -176,5 +188,5 @@ func (m *mock) stopFail(ctx context.Context, extra map[string]interface{}) error
 }
 
 func (m *mock) IsPowered(ctx context.Context, extra map[string]interface{}) (bool, float64, error) {
-	return false, 0, nil
+	return true, 0, nil
```

Comment:
The second value returned is the power percent. Are you sure you want to return `true` ("yes, it's powered") and 0 ("the power level is 0") at the same time?

Reply:
We're not using the power value anywhere, I can add it but the end condition is waiting for a`false` from this mock powered thing.